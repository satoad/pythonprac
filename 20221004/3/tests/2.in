(4,2,7,4,6,87,7,19,22,11,64), (2,54,67,3,2,22,11)