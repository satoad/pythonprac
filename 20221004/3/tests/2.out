(4, 7, 4, 6, 87, 7, 19, 64)
